recipe/meta.yaml

```diff
@@ -35,40 +35,38 @@ requirements:
     - make                                   # [unix]
     - pkg-config
   host:
-    - boost-cpp                              # [win]
-    - libboost
+    # Boost 1.82 has an issue on Windows: `error C2668: 'boost::hash_value': ambiguous call to overloaded function`,
+    # to avoid this we use Boost 1.73 on Windows.
+    - boost-cpp 1.82
+    - libboost 1.82
```

Comment:
should we need only one of them boost-cpp or libboost, but not both ?

Reply:
Yeah, boost-cpp was required for v1.73 on win but for v1.82 it might be not needed.